Develop

dfrancogu opened the pull request.

Carlos-Micro committed:
Initial commit

Carlos-Micro committed:
Test and minor fixes

Carlos-Micro committed:
Code format and test fixes

Carlos-Micro committed:
Refactor code, added json mocks and fixed several bugs

Carlos-Micro committed:
Removed dev code

Carlos-Micro committed:
Changed names on reports.json

Carlos-Micro committed:
Change filter on assets call and data at reports.json.

sergiopalacio committed:
Feature/ETDS-449 Jenkinsfile pipeline

Squashed commit of the following:

commit 9e663c6a2e1040d5cb9194ddb2b127f2bc7aeef9
Author: Sergio Palacio Ródenas <sergio.palaciorodenas@cloudblue.com>
Date:   Mon Jan 9 12:33:48 2023 +0100

    Feature/ETDS-449 Jenkinsfile pipeline

Carlos-Micro committed:
Pull request #2: Develop

Merge in SITC/adobe-reports from develop to master

Squashed commit of the following:

commit 675da5ebf32076b96e0fd2cc3966a3c79bbb4237
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 17:49:12 2023 +0100

    Minor improvements to excel data and README.md

commit 61f73461418cae292ad3fa063f90f7b62b28482f
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 17:20:39 2023 +0100

    Removed useless code, refactored request/entrypoint and improved README.md

commit fadf3b472c61700a9a2eda4b2797ec793e095dff
Merge: faadff1 4a510cb
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 16:24:49 2023 +0100

    Resolve conflicts

commit faadff13cc3e842f33b8c3f67c021827e30baec2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 16:14:40 2023 +0100

    Create Jenkinsfile

commit e97c1f0cd5dc1a6482242404bbf72d49f73f6577
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 15:02:57 2023 +0100

    Refactor test, modified method calculate_renewal_date() and improved test methods.

commit 02bae8e7cb02ae6d40c6ab6fc1bae6934959d9e2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Thu Jan 5 14:14:25 2023 +0100

    Created request report and refactoring several parts of code.
    Added mocks and test for new report.

Carlos-Micro committed:
Pull request #3: Develop

Merge in SITC/adobe-reports from develop to master

Squashed commit of the following:

commit 36f02f9688fd739d3321fcd5bb34bdf9f6f5b269
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:44:21 2023 +0100

    Fixed test requests

commit 9d708fcff9889926d99bc540dde4b0b906400210
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:21:24 2023 +0100

    Deleted unused templates

commit baf1ae15af6795f8e769a1d2d1bf47027bd3e6f1
Merge: 2160e13 93aeae2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:10:33 2023 +0100

    Merge remote-tracking branch 'origin/develop' into develop

commit 2160e134c492b1f5ffcbc6911144fcd4849f3957
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:09:55 2023 +0100

    Fixed xlsx cells, report renderers and delta per item.

commit 93aeae22d17bbd7cf19686d33fe9553eef65b7d7
Merge: 675da5e 4c59b74
Author: David Franco <dfranco00@ESSAN069616LT.local>
Date:   Tue Jan 10 11:00:32 2023 +0100

    Merge branch 'master' into develop

commit 675da5ebf32076b96e0fd2cc3966a3c79bbb4237
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 17:49:12 2023 +0100

    Minor improvements to excel data and README.md

commit 61f73461418cae292ad3fa063f90f7b62b28482f
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 17:20:39 2023 +0100

    Removed useless code, refactored request/entrypoint and improved README.md

commit fadf3b472c61700a9a2eda4b2797ec793e095dff
Merge: faadff1 4a510cb
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 16:24:49 2023 +0100

    Resolve conflicts

commit faadff13cc3e842f33b8c3f67c021827e30baec2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 16:14:40 2023 +0100

    Create Jenkinsfile

commit e97c1f0cd5dc1a6482242404bbf72d49f73f6577
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 15:02:57 2023 +0100

    Refactor test, modified method calculate_renewal_date() and improved test methods.

commit 02bae8e7cb02ae6d40c6ab6fc1bae6934959d9e2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Thu Jan 5 14:14:25 2023 +0100

    Created request report and refactoring several parts of code.
    Added mocks and test for new report.

Commit:
Pull request #4: Develop

Merge in SITC/adobe-reports from develop to master

Squashed commit of the following:

commit f699539e8a1bff3783008fc74831f46cdff037da
Author: carlosa <ZyPP74ncSwPi>
Date:   Thu Jan 12 14:03:10 2023 +0100

    Test items fixed

commit 9eee4d9c1a69ea171fbf8dec42bb2971fd8078d0
Merge: ad9b341 6c64cb4
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Thu Jan 12 13:50:39 2023 +0100

    commit

commit ad9b341f22b05133b4cf5de441f67dac045fe2fb
Author: carlosa <ZyPP74ncSwPi>
Date:   Thu Jan 12 13:35:25 2023 +0100

    Fixed test requests

commit 36f02f9688fd739d3321fcd5bb34bdf9f6f5b269
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:44:21 2023 +0100

    Fixed test requests

commit 9d708fcff9889926d99bc540dde4b0b906400210
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:21:24 2023 +0100

    Deleted unused templates

commit baf1ae15af6795f8e769a1d2d1bf47027bd3e6f1
Merge: 2160e13 93aeae2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:10:33 2023 +0100

    Merge remote-tracking branch 'origin/develop' into develop

commit 2160e134c492b1f5ffcbc6911144fcd4849f3957
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Tue Jan 10 12:09:55 2023 +0100

    Fixed xlsx cells, report renderers and delta per item.

commit 93aeae22d17bbd7cf19686d33fe9553eef65b7d7
Merge: 675da5e 4c59b74
Author: David Franco <dfranco00@ESSAN069616LT.local>
Date:   Tue Jan 10 11:00:32 2023 +0100

    Merge branch 'master' into develop

commit 675da5ebf32076b96e0fd2cc3966a3c79bbb4237
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 17:49:12 2023 +0100

    Minor improvements to excel data and README.md

commit 61f73461418cae292ad3fa063f90f7b62b28482f
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 17:20:39 2023 +0100

    Removed useless code, refactored request/entrypoint and improved README.md

commit fadf3b472c61700a9a2eda4b2797ec793e095dff
Merge: faadff1 4a510cb
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 16:24:49 2023 +0100

    Resolve conflicts

commit faadff13cc3e842f33b8c3f67c021827e30baec2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 16:14:40 2023 +0100

    Create Jenkinsfile

commit e97c1f0cd5dc1a6482242404bbf72d49f73f6577
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Mon Jan 9 15:02:57 2023 +0100

    Refactor test, modified method calculate_renewal_date() and improved test methods.

commit 02bae8e7cb02ae6d40c6ab6fc1bae6934959d9e2
Author: Carlos Anuarbe <Carlos.Anuarbe@ingrammicro.com>
Date:   Thu Jan 5 14:14:25 2023 +0100

    Created request report and refactoring several parts of code.
    Added mocks and test for new report.

Commit:
Pull request #6: CAD-760

Merge in SITC/adobe-reports from CAD-760 to master

Squashed commit of the following:

commit d80e534fe4961003a33fa64dc71bb35e95d78b1c
Author: David Franco <dfranco00@ESSAN069616LT.local>
Date:   Wed Mar 8 18:13:15 2023 +0100

    CAD-760

    remove the .idea

commit 2da61ee0e5684e2f6a5806fd5f53df3f6a591f49
Author: David Franco <dfranco00@ESSAN069616LT.local>
Date:   Wed Mar 8 18:10:59 2023 +0100

    CAD-760

    add changes in the entrypoint

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cloudblue/adobe_reports?pullRequest=4) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/cloudblue/adobe_reports?pullRequest=4) before we can accept your contribution.<br/>**1** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: sergiopalacio<br/>:x: David Franco<br/>:x: indaleciocb<br/>:x: Carlos-Micro<br/><hr/>**David Franco** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cloudblue/adobe_reports?pullRequest=4) it.</sub>

indaleciocb committed:
Pull request #8: Feature/ETDS-531 Rest of MultiBranchPipeline projects

Merge in SITC/adobe-reports from feature/ETDS-531-Rest-of-MultiBranchPipeline-projects to master

Squashed commit of the following:

commit 8ea8aa7c4e8882c027f30cf4d2ba7b7189d8609d
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Jun 7 13:10:45 2023 +0200

    Test

commit ee3b4f54290dcba316742166aa346d56ff68142f
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Jun 7 13:08:23 2023 +0200

    Test

commit f5eba3d1eac213c4e70bddeaa1e1c4b859fb4306
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Jun 7 13:07:16 2023 +0200

    Only package dependencies for requirements.txt

commit 555bb43de9fa0cbaaf12a01af7f93cb52fec12a1
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Jun 7 12:59:44 2023 +0200

    Fix errors.

commit 40e0abdef4a3dcc9e38677d70ed62c0cf3dee7d3
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Jun 7 12:58:48 2023 +0200

    Fix errors

commit 3188bb42df353b5635a1b46b2e9cc046811c0277
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Jun 7 12:54:32 2023 +0200

    Remove old jenkinsfile and create the new one.

indaleciocb committed:
Pull request #9: Add checkmarx and update jenkinsfile.

Merge in SITC/adobe-reports from feature/ETDS-564-Onboard-DevOps-Extensions-into-Checkmarx to master

Squashed commit of the following:

commit d860c2da5a9626603f76ebf6f40476ef7580ae3b
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Jul 12 15:03:27 2023 +0200

    Add checkmarx and update jenkinsfile.

indaleciocb committed:
Pull request #10: Update jenkinsfile.

Merge in SITC/adobe-reports from feature/ETDS-600-Update-jenkinsfiles-to-upload-files-to-jfrog to master

Squashed commit of the following:

commit 4c4074285982c1ac4eac3954f6f5db43c4b61a23
Author: indaleciocb <indalecio.sierra@cloudblue.com>
Date:   Wed Aug 9 10:55:27 2023 +0200

    Update jenkinsfile.

Commit:
Pull request #11: CAD-1153 include the filters and templates to show the 3YC assets and request

Merge in SITC/adobe-reports from CAD-1153-p1-find-in-the-active-assets-reports-the-new-columns to master

Squashed commit of the following:

commit 69174ce336cb2cd9b674f2b47bb45b94449c74c3
Author: David Franco <david.franco@cloublue.com>
Date:   Thu Aug 17 16:51:22 2023 +0200

    CAD-1153 improve test coverage

commit 7dbd3a53b3c7251ef5c5cead2a0a28a70e7d6337
Author: David Franco <david.franco@cloublue.com>
Date:   Thu Aug 17 13:56:25 2023 +0200

    CAD-1153 improve the test coverage

commit f173a708e35dcb63b6bf419995fd98ed1d458c95
Author: David Franco <david.franco@cloublue.com>
Date:   Thu Aug 17 13:42:11 2023 +0200

    CAD-1153 correct the test

commit 167d06753ba8acccb8e5a5b63b9e72da10495abd
Author: David Franco <david.franco@cloublue.com>
Date:   Thu Aug 17 13:32:38 2023 +0200

    CAD-1153 include the filters and templates to show the 3YC assets and request

Commit:
Pull request #12: CAD-1153  correct some method to recover the information

Merge in SITC/adobe-reports from CAD-1153-p1-find-in-the-active-assets-reports-the-new-columns to master

Squashed commit of the following:

commit d47c243f742952c9fd4c7109938ec3cce0f35f0b
Author: David Franco <david.franco@cloublue.com>
Date:   Fri Aug 18 08:33:53 2023 +0200

    CAD-1153  correct some method to recover the information
